seeder/src/renderer/router.ts

```diff
@@ -0,0 +1,43 @@
+import { RouteDefinition } from "@solidjs/router";
+import { lazy } from "solid-js";
+
+export const routes: RouteDefinition[] = [
+	{
+		path: "/",
+		component: lazy(() => import("@layouts/RootLayout")),
+		children: [
+			{
+				path: "",
+				component: lazy(() => import("@routes/home"))
+			},
+			{
+				path: "/get-character-urls",
+				component: lazy(() => import("@routes/character"))
+			},
+			{
+				path: "/get-demographics",
+				component: lazy(() => import("@routes/demographics"))
+			},
+			{
+				path: "/get-staff-urls",
+				component: lazy(() => import("@routes/staff"))
+			},
+			{
+				path: "/get-anime-explicit-genres",
+				component: lazy(() => import("@routes/anime/explicit-genres"))
+			},
+			{
+				path: "/get-anime-genres",
+				component: lazy(() => import("@routes/anime/genres"))
+			},
+			{
+				path: "/get-anime-urls",
+				component: lazy(() => import("@routes/anime"))
+			},
+			{
+				path: "/get-anime-themes",
+				component: lazy(() => import("@routes/anime/themes"))
+			}
+		]
+	}
```

Comment:
Very unncessary. Make it in `main.tsx`

Reply:
THis might become become huge you know that right?

Reply:
Yes. i know and thats why i said so

Reply:
Resolved on https://github.com/coreproject-moe/CoreProject/pull/3918/commits/012dc073d3491ac68bd5cde187e2c39999b55c92